app/consumer-democracy/app.go

```diff
@@ -517,11 +522,11 @@ func New(
 		vestingtypes.ModuleName,
 		ibctransfertypes.ModuleName,
 		ibchost.ModuleName,
-		consumertypes.ModuleName,
 	)
 	app.MM.SetOrderEndBlockers(
 		crisistypes.ModuleName,
 		govtypes.ModuleName,
+		consumertypes.ModuleName, // Note: consumer endblocker before staking module
```

Comment:
@jstr1121 Why is this order necessary?

Reply:
With https://github.com/cosmos/interchain-security/pull/757/commits/1269163b9c75885ac4df83914b5e166a79de6a62, the order doesn't matter AFAIK